desktop/transactionManager.js

```diff
@@ -88,6 +92,20 @@ class TransactionManager {
     return { transactions: this.transactions };
   };
 
+  updateTxContact = ({ newData, accountIndex }) => {
+    const { address, nickname } = newData;
+    const addrString = address.substring(2).toLowerCase();
```

Comment:
make sure that address has `0x` prefix before slicing it

Reply:
0x must be included to pass validity check to form address length of 42. just thought of something, instead of "const { address, nickname }" can we do "let { address, nickname }"? then we don't need addrString

Reply:
this will cause lint error for sure, cause nickname doesn't mutate